presidio-anonymizer/tests/integration/test_deanonymize_engine.py

```diff
@@ -20,7 +20,7 @@ def test_given_operator_decrypt_with_valid_params_then_decrypt_text_successfully
     decryption = engine.deanonymize(
         text,
         encryption_results,
-        {"DEFAULT": OperatorConfig(Decrypt.NAME, {"key": "WmZq4t7w!z%C&F)J"})},
+        {"DEFAULT": OperatorConfig(Decrypt.NAME, {"key": b'WmZq4t7w!z%C&F)J'})},
```

Comment:
**nit**: Should the single quotes be double quotes to be consistent with other quotes throughout the file?

Reply:
maybe, I am used to seeing byte string represented with single quotes, not sure what is the convention there, but all bytestrings in the repo are with single quotes so it is consistent